Resources/translations/SonataArticleBundle.en.xliff

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="0" resname="article">
```

Comment:
this is from the admin bundle:
```xml
        <body>
            <trans-unit id="sonata_administration">
                <source>sonata_administration</source>
                <target>Administration</target>
            </trans-unit>
            <trans-unit id="action_delete">
                <source>action_delete</source>
                <target>Delete</target>
            </trans-unit>
```

please remove the `resname`-attribute and use the value as the value for the `id`-attribute!

Thank you 👍